elements/lisk-chain/src/index.ts

```diff
@@ -17,7 +17,7 @@ const events = { EVENT_DELETE_BLOCK, EVENT_NEW_BLOCK, EVENT_VALIDATORS_CHANGED }
 
 export { events };
 export { Chain } from './chain';
-export { Transaction, transactionSchema, calculateMinFee } from './transaction';
+export { Transaction, TransactionInput, transactionSchema, calculateMinFee } from './transaction';
```

Comment:
why do we need to export the input `TransactionInput`? i do not see it being used anywhere, can we not export if it's not used?

Reply:
Sure I originally was using it but I updated the code to not use it. Will un-export.